maas/region.sls

```diff
@@ -125,11 +126,14 @@ maas_region_services:
     - cmd: maas_region_syncdb
   - watch:
     - file: /etc/maas/regiond.conf
+  {%- if grains.get('kitchen-test') %}
+  - onlyif: /bin/false
+  {%- endif %}
 
 maas_region_syncdb:
   cmd.run:
   - names:
-    - maas-region syncdb
+    - maas-region syncdb --noinput
```

Comment:
Mind this, during CI testing up/down I have some strange situation that syncdb asks for input, this is the correct approach for  formula